tensorflow/buildspec-tf1.yml

```diff
@@ -0,0 +1,77 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+  region: &REGION <set-$REGION-in-environment>
+  framework: &FRAMEWORK tensorflow
+  version: &VERSION 1.15.2
+  cuda_version: &CUDA_VERSION cu101
+  os_version: &OS_VERSION ubuntu18.04
+
+  repository_info:
+    training_repository: &TRAINING_REPOSITORY
+      image_type: &TRAINING_IMAGE_TYPE training
+      root: !join [ *FRAMEWORK, "/", *TRAINING_IMAGE_TYPE ]
+      repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *TRAINING_IMAGE_TYPE]
+      repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+    inference_repository: &INFERENCE_REPOSITORY
+      image_type: &INFERENCE_IMAGE_TYPE inference
+      root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+      repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE]
+      repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+  context:
+    training_context: &TRAINING_CONTEXT
+      dockerd-entrypoint:
+        source: docker/build_artifacts/dockerd-entrypoint.py
+        target: dockerd-entrypoint.py
+      deep_learning_container:
+        source: docker/build_artifacts/deep_learning_container.py
+        target: deep_learning_container.py
+    inference_context: &INFERENCE_CONTEXT
+      sagemaker_package_name:
+        source: docker/build_artifacts/sagemaker
+        target: sagemaker
+      deep_learning_container:
+        source: docker/build_artifacts/deep_learning_container.py
+        target: deep_learning_container.py
+      init:
+        source: docker/build_artifacts/__init__.py
+        target: __init__.py
+      dockerd-entrypoint:
+        source: docker/build_artifacts/dockerd-entrypoint.py
+        target: dockerd-entrypoint.py
+
+  images:
+    BuildTensorflowCPUTrainPy3DockerImage:
```

Comment:
currently it looks like the tf1 inference jobs are not added here -- we should add them back. Also py2 images if we have them

Reply:
Nice point , I just copied TF2 buildspec , I will add those.